src/components/chip-set/chip-set.tsx

```diff
@@ -323,7 +323,9 @@ export class ChipSet {
                         'mdc-text-field--disabled':
                             this.readonly || this.disabled,
                         'mdc-text-field--required': this.required,
-                        'force-float': !!(this.value.length || this.editMode),
+                        'mdc-floating-label--float-above': !!(
+                            this.value.length || this.editMode
+                        ),
```

Comment:
I think this might also mess stuff up. When the picker has a value the label looks fine. If I focus the element and then blur it again, the label will overlap with the chip

Reply:
> I think this might also mess stuff up. When the picker has a value the label looks fine. If I focus the element and then blur it again, the label will overlap with the chip

Ahh thought I tested everything but yes you're right...